build-system/pr-check/experiment-tests.js

```diff
@@ -60,6 +61,8 @@ function build_(config) {
 }
 
 function test_() {
+  process.env['PERCY_TOKEN'] = atob(process.env.PERCY_TOKEN_ENCODED);
+  timedExecOrDie('gulp visual-diff --nobuild --master');
```

Comment:
This approach will end up overriding the baseline for visual diffs with one of the experiment builds. `--master` wouldn't work here. I think instead we should have multiple side-projects for the experiments - that is, build these into an `ampproject/amphtml-exp-a` [`-b`, `-c`] projects on Percy. I think Percy supports sending emails for broken builds, so we can do that.
That'll require setting up different tokens for each of these builds, and using those tokens instead of the existing `PERCY_TOKEN_ENCODED`

Reply:
Shouldn't the baseline for an RTV experiment be the same as master?

Reply:
Hmm... yeah, probably 🙃
I'm not sure how to get it to not override the master baseline (because this is built on the master branch) and how to report those builds separately

Reply:
But you're right that we don't want to override the master baseline. It's not guaranteed that an experiment baseline will be the same. Having separate Percy projects might be the way to go but we'd need to sync experimental baselines with master's.

About reporting the builds - my first idea is to set up a router in each Travis experiment job to listen to a Percy `build_approved` web hook. Thoughts?

Reply:
We used to have a simple web server that you can poll to ask whether a build has finished (@rsimha wrote it) - I'm not sure how to retrieve the build number from the Percy agent, but it's probably not too hard :) You can go the web hook way of course but that could be a bit of an overkill. @rsimha for extra thoughts

Reply:
> I'm not sure how to retrieve the build number from the Percy agent, but it's probably not too hard

The web server I wrote exposes only a subset of available fields from a Percy build. It's definitely possible to add the fields you're interested in and re-deploy the webserver. (Will sync up offline once I do some homework on this.)